packages/web/src/pages/_app.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react'
 import App, { AppInitialProps } from 'next/app'
 import { WithApolloProps } from 'next-with-apollo'
 import Head from 'next/head'
-import withApollo from 'src/fixtures/withApollo'
+// import withApollo from 'src/fixtures/withApollo'
```

Comment:
This hoc make that all pages are rendered on server side.
So, vercel deploy it as serverless functions (running on lambda).